docs/mixins/state.md

```diff
@@ -154,7 +155,7 @@ implementations of `modelEvents` and `collectionEvents` in the [Marionette.View]
 ### Setting State
 
 `StateMixin` has a `setState` method that exposes the `Backbone.Model.set`
-for the `StateClass`'s attached `StateModel`.  Implementation will match [Backbone.Model.set](http://backbonejs.org/#Model-set) documentation.
+for the `StateMixin`'s attached `StateModel`.  Implementation will match [Backbone.Model.set](http://backbonejs.org/#Model-set) documentation.
```

Comment:
Yes...and in my search I found that the core `readme.md` had not been updated. 😞